src/containers/MainContainer.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from "react";
+import RestaurantList from "../components/Main/RestaurantList/RestaurantList";
+import CategoryFilter from "../components/Main/CategoryFilter";
+import { RESTAURANTS_DATA } from "../components/Constants/restaurantData";
+
+const MainContainer = () => {
+  const [category, setCategory] = useState("");
+  const filteredRestaurants =
+    category === "" || category === "전체"
+      ? RESTAURANTS_DATA
+      : RESTAURANTS_DATA.filter(
+          (restaurant) => restaurant.category === category
+        );
+
+  return (
+    <main>
+      <CategoryFilter category={category} onChangeCategory={setCategory} />
```

Comment:
여기서 onChangeCategory로 props명을 선정하신 이유가 있을까요? 전 setter함수들을 props로 보낼 때 그 setter 함수 이름 그대로 넘기는 것을 선호해요! 

또한 category라고 하기보다는 selectedCategory라고 네이밍하는 편이 더 가독성에 도움될 것 같습니다.
```suggestion
      <CategoryFilter selectedCategory={selectedCategory} setselectedCategory={setselectedCategory} />
```

Reply:
[https://edu.nextstep.camp/s/JvAf84Zi/ls/03qL4P6Y](url)
2단계의 예시를 그대로 사용했습니다..ㅎㅎ
`CategoryFilter`에서 `Category`를 선택하면, 사용자로부터 `event`를 받아 `CategoryFilter` 내부에서 category를 변경하기 때문에 onChange라고 네이밍하는 것이 괜찮다고 납득했습니다!
그런데 범수님 코드를 보니 setter코드 그대로 넘기는 것이 서로 다른 파일에서 쓸 때 훨씬 직관적이고 일관된 느낌이 드는 것 같습니다!
수정했습니다!